### PR TITLE
Reorder Select, MultiSelect field 515

### PR DESCRIPTION
**Pull request by saravanan-inc**
Add Reorder Select and Multi-select Field using react-dnd

### CODE REVIEW

File: src/components/fields/SingleSelect/Settings.tsx [hunk 1]
```diff
@@ -42,7 +114,18 @@ export default function Settings({ onChange, config }: ISettingsProps) {
...
+            <DraggableSettingsCard
+              options={options}
+              option={option}
+              index={index}
+              onChange={onChange}
+            />
+          ))}
+        </DndProvider>
+
+        {/* {options?.map((option: string) => (
```
Comment (shamsmosowi) [inline src/components/fields/SingleSelect/Settings.tsx:128]: you can remove the old code here, instead of commenting it out

**Review by shamsmosowi — CHANGES_REQUESTED** [review]
Hi @saravanan-inc, thanks for the contribution
the reordering is functional, however to improve it and make it intuitive to users, you can look into the dnd implementation by @iamanishroy, interms of both the icon to the left and the animation during the interaction

https://user-images.githubusercontent.com/13803906/208837692-64f04882-57e3-4c35-aef4-3081d739d67e.mov

let me know if you need an further clarification

File: src/components/fields/SingleSelect/Settings.tsx (hunk 1 above)
Comment (saravanan-inc) [reply src/components/fields/SingleSelect/Settings.tsx:128]: @shamsmosowi  Ok let me check  and update it

**Comment by harinij** [comment]
Hi @saravanan-inc 👋 thanks for working on the feedback, just checking if you need any additional inputs and if there is any update.

File: src/components/fields/SingleSelect/Settings.tsx (hunk 1 above)
Comment (saravanan-inc) [reply src/components/fields/SingleSelect/Settings.tsx:115]: @shamsmosowi  fixed the above mentions, kindly check and let me know if everything is needed

**Comment by saravanan-inc** [comment]
> Hi @saravanan-inc 👋 thanks for working on the feedback, just checking if you need any additional inputs and if there is any update.

@harinij updated the code

File: src/components/fields/SingleSelect/Settings.tsx (hunk 1 above)
Comment (shamsmosowi) [reply src/components/fields/SingleSelect/Settings.tsx:115]: Hi @saravanan-inc , i see you've removed the commented code, but the interaction highlighted in the video is still not updated

Comment (saravanan-inc) [reply src/components/fields/SingleSelect/Settings.tsx:115]: Hi @shamsmosowi the interaction in too updated could you please elaborate the issue ?

Comment (saravanan-inc) [reply src/components/fields/SingleSelect/Settings.tsx]: The draggable icon and the animation are already updated in the previous commit, please check the code.

**Review by shamsmosowi — APPROVED** [review]
looks great 👍

**Pull request merged**